src/seeds.proposals.cpp

```diff
@@ -301,6 +328,8 @@ void proposals::onperiod() {
     while (pitr != props.end()) {
       uint64_t prop_id = pitr -> id;
 
+      print("running proposal: ", pitr->id, "\n");
+
       // active proposals are evaluated
```

Comment:
I think we need to merge master into this to get the new quorum calculations

Reply:
Yes, I updated it